openinfra_core/src/main/java/de/btu/openinfra/backend/db/daos/AttributeValueDao.java

```diff
@@ -173,14 +180,14 @@ public AttributeValuePojo mapToPojo(Locale locale, AttributeValue av) {
 
 			} // end switch
 
-			avPojo.setAttributeTypeId(
+			pojo.setAttributeTypeId(
 					av.getAttributeTypeToAttributeTypeGroup()
 					.getAttributeType()
 					.getId());
-			avPojo.setUuid(id);
-			avPojo.setTrid(av.getXmin());
-			avPojo.setAttributeValueType(type);
-			return avPojo;
+			pojo.setUuid(id);
+			pojo.setTrid(av.getXmin());

```

Comment:
It exists a contructor which consumes the uuid and the trid. This constructor should be used.